tests/functional/test_override_database/test_override_database.py

```diff
@@ -31,6 +29,12 @@ def project_config_update(self):
             }
         }
 
+    @pytest.fixture(scope="class")
+    def clear_test_schema(self, project):
+        yield
+        relation = project.adapter.Relation.create(database=os.getenv("BIGQUERY_TEST_ALT_DATABASE"), schema=project.test_schema)
```

Comment:
Looks like we call `os.getenv("BIGQUERY_TEST_ALT_DATABASE")` in a bunch of places, could we just assign this to a static var?

Reply:
sure